### DISCUSSION TITLE
How to Compile SFC on runtime?

### DISCUSSION THREAD

**Discussion by noor-tg** [nuxt/nuxt#25705] [Questions] [answered]
Hey,
I have a use case (like cms) . When I want to allow users to make their own themes using vue `Single File Component`. So my question is:
**how to compile SFC on runtime to html** ?

**Comment by huang-julien** [comment, marked as answer]
Hey :wave: if you want the runtime bundle of vue, you have to set `vue.runtimeCompiler` to `true` in your nuxt config

**Reply by noor-tg** [reply]
does this work with ssr ?

**Reply by huang-julien** [reply]
yes
